sample/README.md

```diff
@@ -0,0 +1,51 @@
+# Sample
+
+The Sample describes several demos of quickly spinning up different Prebid Server instances with various preset configurations. These samples are intended for audiences with little knowledge about Prebid Server and plan to play around with it locally and see how it works.
+
+# Installation
+
+In the Sample, we use `docker` and `docker-compose` to instantiate examples; with docker providing a unified setup and interface,  you can spin up a demo server instance locally with only one command without knowing all the complexities.
+The docker image used in `docker-compose.yml` is the `Dockerfile` residing in the root level of the repository. 
+
+## Option 1 - Standard Docker Engine
+Install `docker` and `docker-compose` via the [official docker page](https://docs.docker.com/compose/install/#scenario-one-install-docker-desktop). If you cannot use the official docker engine due to restrictions of its license, see the option below about using Podman instead of Docker. 
+
+## Option 2 - Podman
+From MacOS, you can use [podman](https://podman.io/) with these additional steps:
+
+```sh
+$ brew install podman docker-compose
+$ podman machine init
+$ podman machine set --rootful
+$ podman machine start
+$ cd sample
+$ docker-compose up <number>_<name>
```

Comment:
After running `docker compose build --no-cache` the message `naming docker.io/library/pbs-sample` was displayed at the end. Can we clarify that `001_banner` is the `<number>_<name>` want here? Because, from the ouput of the previous command, it could be interpreted to be:
```
docker-compose up 001_docker.io/library/pbs-sample
```
which is incorrect.

Reply:
`001_banner` is the container instance name
`pbs-sample `is the image name

Reply:
i have purged all my images and containers and have a refresh start on the docker


here is the command i executed
1. `docker compose build --no-cache`
2. `docker compose up 001_banner`

i have no issue spinning up a running 001_banner container, i can not reproduce your issue

Reply:
@VeronikaSolovei9 do you encountered the issue @guscarreon encountered, i try to understand if the issue is unique on @guscarreon or universal